src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -136,10 +136,18 @@ export const AccessApprovalsTable: React.FunctionComponent<
           })}
         </tbody>
       </Table>
+      {isLoading && (
+        <div className="SRC-center-text">
+          <SynapseSpinner size={40} />
+        </div>
+      )}
```

Comment:
Since `isLoading` is only true when we we have no data and are fetching more, it can be pulled out of a complex expression.